Formula/semgrep.rb

```diff
@@ -4,8 +4,8 @@ class Semgrep < Formula
   desc "Easily detect and prevent bugs and anti-patterns in your codebase"
   homepage "https://semgrep.dev"
   url "https://github.com/returntocorp/semgrep.git",
-    tag:      "v0.27.0",
-    revision: "549bfa79f81939ac06e57a527b5f0d46c87c46ce"
+    tag:      "v0.28.0",
+    revision: "bc9375f8d42bbf89a715638894d58e87da974748"
```

Comment:
```suggestion
    revision: "e5670ed176e18a6927fea85d2a82212b5d272806"
```

Needs bump after https://github.com/returntocorp/semgrep/pull/1862/commits/e5670ed176e18a6927fea85d2a82212b5d272806